Sources/TokamakDOM/Storage/LocalStorage.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/20/20.
+//
+
+import JavaScriptKit
+import OpenCombine
+import TokamakCore
+
+public class LocalStorage: WebStorage, _AppStorageProvider {
+  let storage = JSObjectRef.global.localStorage.object!
+
+  required init() {
+    _ = JSObjectRef.global.window.object!.addEventListener!("storage", JSClosure { _ in
+      self.publisher.send()
+      return .undefined
+    })
```

Comment:
Shouldn’t this be tracked for `SessionStorage` too and differentiate between them by checking `event.storageArea`?

Reply:
It's not necessary, as `SceneStorage` is only for the current tab, we can track it entirely with `OpenCombine` (the `rootPublisher` does this). It's needed for `AppStorage` because another instance could update it, and the only way to be notified of that change is via Javascript.

Reply:
`rootPublisher` is also used on `LocalStorage` since the `storage` event is only sent when other tabs update the storage.